Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -1517,6 +1532,7 @@ public static void CreateUpdateRunSetFromJSON(RunsetExecutor runsetExecutor, Gin
                 runSetConfig.SelfHealingConfiguration.AutoUpdateApplicationModel = dynamicRunsetConfigs.SelfHealingConfiguration.AutoUpdateApplicationModel;
                 runSetConfig.SelfHealingConfiguration.SaveChangesInSourceControl = dynamicRunsetConfigs.SelfHealingConfiguration.SaveChangesInSourceControl;
                 runSetConfig.SelfHealingConfiguration.ReprioritizePOMLocators = dynamicRunsetConfigs.SelfHealingConfiguration.ReprioritizePOMLocators;
+                runSetConfig.SelfHealingConfiguration.AutoExecuteInSimulateionMode = dynamicRunsetConfigs.SelfHealingConfiguration.AutoExecuteInSimulateionMode;
```

Comment:
Please correct spelling of SImulation

Reply:
Done from Ginger side, need to update nuget as well